src/main/resources/application.yaml

```diff
@@ -145,16 +145,16 @@ validations:
     SIAC_WEEKLY_HEARING_LIST: "schemas/non-strategic/siac_weekly_hearing_list.json"
     POAC_WEEKLY_HEARING_LIST: "schemas/non-strategic/siac_weekly_hearing_list.json"
     PAAC_WEEKLY_HEARING_LIST: "schemas/non-strategic/siac_weekly_hearing_list.json"
-    FFT_TAX_WEEKLY_HEARING_LIST: "schemas/non-strategic/fft-tax-chamber-tribunal-weekly-hearing-list.json"
-    FFT_LR_WEEKLY_HEARING_LIST: "schemas/non-strategic/fft-land-registry-tribunal-weekly-hearing-list.json"
+    FTT_TAX_WEEKLY_HEARING_LIST: "schemas/non-strategic/ftt-tax-chamber-tribunal-weekly-hearing-list.json"
+    FTT_LR_WEEKLY_HEARING_LIST: "schemas/non-strategic/ftt-land-registry-tribunal-weekly-hearing-list.json"
```

Comment:
You updated the locations of FTT file name here but are the actual JSON file names updated?

Reply:
Done. Don't know how I forgot it :)